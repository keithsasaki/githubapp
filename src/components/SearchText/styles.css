@font-face {
    font-family: 'Monaco';
    src: local('Monaco'), url(../../assets/fonts/Monaco.ttf) format('ttf');
}

.github-search {
    font-family: 'Monaco';
    color: #000000;
}

.github-search .text-style-1 {
    font-family: Raleway;
    font-weight: 200;
    font-style: italic;
}

@media (max-width: 700px){
    .github-search {
        font-family: 'Monaco';
        font-size: 4rem;
        color: #000000;
        margin-right: 2rem;
    }
    
    .github-search .text-style-1 {
        font-family: Raleway;
        font-weight: 200;
        font-style: italic;
    }


}