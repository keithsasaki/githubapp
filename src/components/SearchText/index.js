import React from 'react';
import './styles.css';

function SearchText() {
    return(
        <div className="github-search">
            <div id="text">Github <span className="text-style-1">Search</span></div>
        </div>       
    )
}

export default SearchText;