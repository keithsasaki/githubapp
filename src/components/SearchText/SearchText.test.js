import React from 'react';
import '@testing-library/jest-dom';

import renderer from 'react-test-renderer';
import SearchText from '.';

describe("Testing SearchText Component", () => {
    it("matches snapshot", () => {
        const tree = renderer.create(<SearchText />).toJSON();
        expect(tree).toMatchSnapshot();
    });
})