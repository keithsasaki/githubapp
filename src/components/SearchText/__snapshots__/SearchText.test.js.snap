// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing SearchText Component matches snapshot 1`] = `
<div
  className="github-search"
>
  <div
    id="text"
  >
    Github 
    <span
      className="text-style-1"
    >
      Search
    </span>
  </div>
</div>
`;
